lcls-twincat-pmps/PMPS/DUTs/UserDUT/ST_BeamParams.TcDUT

```diff
@@ -10,19 +10,19 @@
 			field: LOPR 0;
 			field: PREC 2;
         '}
-		nTran : REAL := 1;
+		nTran : REAL := 0;
         (* Pulse-rate *)
         {attribute 'pytmc' := 'pv: Rate
             io: i
             field: EGU Hz
         '}
-        nRate : UDINT := 120;
+        nRate : UDINT := 0;
         (* Photon energy ranges *)
         {attribute 'pytmc' := 'pv: PhotonEnergyRanges
             io: i
             field: EGU eV'}
         {attribute 'displaymode' := 'binary'}
-        neVRange : DWORD := 2#1111_1111_1111_1111_1111_1111_1111_1111;
+        neVRange : DWORD := 0;//2#1111_1111_1111_1111_1111_1111_1111_1111;
```

Comment:
Do you expect we'll need to change this back ever? If not, might as well just make it very clear by removing the comment.
```suggestion
        neVRange : DWORD := 0;
```